src/transformers/models/llava_next/processing_llava_next.py

```diff
@@ -199,8 +198,8 @@ def _get_unpadded_features(self, height, width, patches_height, patches_width, s
         because it divided each image into patches depending on its resolution. Therefore we need to calculate how many
         patches an image is divided into and get the number of features from that.
         """
-        current_width = patches_height * scale_height
-        current_height = patches_width * scale_width
+        current_height = patches_height * scale_height
+        current_width = patches_width * scale_width
```

Comment:
The order was corrected in other PR for modeling code but I forgot to add same correction to the processing logic